diff --git a/sbmaker/sbhistograms.py b/sbmaker/sbhistograms.py
index 30c9089..be0f2a7 100644
--- a/sbmaker/sbhistograms.py
+++ b/sbmaker/sbhistograms.py
@@ -10,8 +10,8 @@ class SBHistograms():
         self._histograms    = dict()
         self.root_file_name = root_file_name
         self.region_prefix  = region_prefix
-        self._disc = "effmHH"
-        self._signal_prefix = "Hhhbbtautau"
+        self._disc = "subsmhh"
+        self._signal_prefix = "Xtohh"
         self.masses         = masses
         self.store          = store
         self._get_histograms()
@@ -33,19 +33,19 @@ class SBHistograms():
                 if self._signal_prefix in name and self._signal_prefix+str(mass) not in name: continue
                 if self._signal_prefix+str(mass)+"Py8" in name: continue
                 name_list.append(name)
-        root_file.cd("Systematics")
-        hash = gDirectory.GetListOfKeys()
-        iter = hash.MakeIterator()
-        key = iter.Next()
-        while key:
-            name = key.ReadObj().GetName()
-            key = iter.Next()
-            if self.region_prefix + "_" + self._disc in name:
-                if self._signal_prefix in name and self._signal_prefix+str(mass) not in name: continue
-                if self._signal_prefix+str(mass)+"Py8" in name: continue
-                if "data" in name and self._disc + "_Sys" in name: continue
-                if name.split('_Sys')[0] not in name_list: continue # some inconsistency of nomial and systs
-                name_list.append(name)
+        # root_file.cd("Systematics")
+        # hash = gDirectory.GetListOfKeys()
+        # iter = hash.MakeIterator()
+        # key = iter.Next()
+        # while key:
+        #     name = key.ReadObj().GetName()
+        #     key = iter.Next()
+        #     if self.region_prefix + "_" + self._disc in name:
+        #         if self._signal_prefix in name and self._signal_prefix+str(mass) not in name: continue
+        #         if self._signal_prefix+str(mass)+"Py8" in name: continue
+        #         if "data" in name and self._disc + "_Sys" in name: continue
+        #         if name.split('_Sys')[0] not in name_list: continue # some inconsistency of nomial and systs
+        #         name_list.append(name)
         root_file.Close()
         return name_list
 
diff --git a/sbmaker/sbyields.py b/sbmaker/sbyields.py
index e9cb9f4..d039611 100644
--- a/sbmaker/sbyields.py
+++ b/sbmaker/sbyields.py
@@ -31,8 +31,8 @@ class SBYields():
         self._n_bins = len(self.binning) - 1
         self.root_file_name = root_file_name
         self.region_prefix = region_prefix
-        self._disc = "effmHH"
-        self._signal_prefix = "Hhhbbtautau"
+        self._disc = "subsmhh"
+        self._signal_prefix = "Xtohh"
         self.histograms = histograms if not load else self._load_histograms()
         self.load = load
         self.store = store
@@ -219,6 +219,9 @@ class SBYields():
                         nEvents, _ = self._histogram_info(root_file, name)
                         if self._signal_prefix not in process and 'data' not in process:
                             total_bkg += sum(nEvents)
+            yields_mass["data"] = {}
+            yields_mass["data"]["nEvents"] = [total_bkg] # HC, this means always use 1 bin !
+            yields_mass["data"]["nEventsErr"] = [sqrt(total_bkg)] # HC, this means always use 1 bin !
             print("mass {} bkg {}".format(mass, total_bkg))
             for process, name_list in name_dict.items():
                 print("-> processing {} / {} ... ".format(mass, process))
diff --git a/test/test.py b/test/test.py
index dd8e112..3329cde 100644
--- a/test/test.py
+++ b/test/test.py
@@ -12,7 +12,7 @@ yields = None
 with open('/Users/bowen/PycharmProjects/SampleBuilderMaker/pickle_files/yields.dictionary', 'rb') as yields_pickle:
     yields = pickle.load(yields_pickle)
 
-signal_prefix = "Hhhbbtautau"
+signal_prefix = "Xtohh"
 
 color_dict = {"Zbb": kAzure, "Zbc": kAzure, "Zbl": kAzure,
               "Zcc": kAzure, "Zcl": kBlue, "Zl": kBlue,
@@ -119,4 +119,4 @@ def print_syst_table(mass):
 for mass in [1000, 1200, 1400, 1600, 1800, 2000, 2500, 3000]:
     print("@ {}: ".format(mass))
     print_info(mass)
-    print_syst_table(mass)
+    # print_syst_table(mass)
diff --git a/test/test_histograms.py b/test/test_histograms.py
index 96639c9..cf3a356 100644
--- a/test/test_histograms.py
+++ b/test/test_histograms.py
@@ -5,8 +5,8 @@ from ROOT import gROOT
 print(gROOT.GetVersion())
 from sbmaker.sbhistograms import SBHistograms
 
-filename = "/Users/bowen/Documents/work/Boosted/root_files/submitDir_v10_w21_u3_mc16ade.root"
-prefix = "2tag2pjet_0ptv_SRLRJM60160"
+filename = "/Users/bowen/Documents/work/Boosted/root_files/readeroutput_2dmasswindows.root"
+prefix = "2tag2pjet_0ptv_SR"
 masses = [1000, 1200, 1400, 1600, 1800, 2000, 2500, 3000]
 
 sbh = SBHistograms(filename, prefix, masses, True)
diff --git a/test/test_yields.py b/test/test_yields.py
index a6d9716..dc8b16a 100644
--- a/test/test_yields.py
+++ b/test/test_yields.py
@@ -5,9 +5,9 @@ from ROOT import gROOT
 print(gROOT.GetVersion())
 from sbmaker.sbyields import SBYields
 
-filename = "/Users/bowen/Documents/work/Boosted/root_files/submitDir_v10_w21_u3_mc16ade.root"
-prefix = "2tag2pjet_0ptv_SRLRJM60160"
-binning = [0., 1400., 4000.]
+filename = "/Users/bowen/Documents/work/Boosted/root_files/readeroutput_2dmasswindows.root"
+prefix = "2tag2pjet_0ptv_SR"
+binning = [0., 4000.]
 useCache = True
 
 sby = SBYields(filename, prefix, binning, None, useCache, True)
